curriculum/challenges/_meta/learn-basic-dom-manipulation-by-building-a-role-playing-game/meta.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "Learn Basic JavaScript By Building a Role Playing Game",
+  "name": "Learn Basic DOM Manipulation by Building a Role Playing Game",
   "isUpcomingChange": false,
   "usesMultifileEditor": true,
   "hasEditableBoundaries": true,
-  "dashedName": "learn-basic-javascript-by-building-a-role-playing-game",
+  "dashedName": "learn-basic-dom-manipulation-by-building-a-role-playing-game",
   "helpCategory": "JavaScript",
   "order": 0,
```

Comment:
TODO:

don't forget to change this to 1 since the intro project will be order 0